src/angular/4-adding-routing/adding-routing.md

```diff
@@ -81,7 +81,8 @@ Create routes for the `HomeComponent` and `RestaurantComponent`. When the route
 Notice that you will be able to click the **Choose a Restaurant** button
 after implementing the solution:
 
-![Choose a restaurant routing](../static/img/angular/4-adding-routing/after.gif 'Choose a restaurant routing')
+<img alt="Choose a restaurant routing." src="../static/img/angular/4-adding-routing/after.gif"
+  style="border: solid 1px black;" width="90%"/>
```

Comment:
Why was this set to 90% and not 100%?

Reply:
hmm I thought I set the width with similar widths in Angular. I guess not.

Reply:
I added requirements to the issue: https://bitovi.atlassian.net/browse/LD-204

Please let me know if you have any questions, I’m happy to talk through it more.